flutter/news/lib/features/frontpage/domain/entities/article.dart

```diff
@@ -1,16 +1,20 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
 import 'package:news/features/frontpage/domain/entities/source.dart';
+part 'article.g.dart';
 
-class Article {
+@JsonSerializable()
+class Article extends Equatable {
   final Source source;
-  final String? author;
+  final String author;
```

Comment:
@Diogo00dev would you mind, based on what you know about the API, select the and mark as `nullable` the properties that could be nullable?